### ISSUE TITLE
Add support for Today widget

### ISSUE THREAD

**Issue by pdileonardo** [sebbu/iControl-Web#24]
It would be great to also have a section in the config file to set up a Today widget.  The app is incredibly useful as is and about the only option I've found that supports Apple Watch, but widget support would make it even better.

**Comment by sebbu** [comment]
Yes. Today widget support would be great and is on the feature list. But I currently do not have much time for this app.
